### PR TITLE
Auto PR due to push to staging

### PR DESCRIPTION
**Pull request by ExC0d3**
Fixed broken API source code test.

### CODE REVIEW

File: capital/trade_utils.py [hunk 1]
```diff
@@ -192,9 +192,13 @@ def get_rand_good(goods_dict, nonzero=False):
             # we can't allocate what we don't have!
             print("Goods are depleted!")
             return None
-
+        # lst = [good for good in
+        #        goods_dict if goods_dict[good][AMT_AVAIL] > 0]
+        # if (len(lst) == 0):
+        #     return None
```
Comment (ExC0d3) [inline capital/trade_utils.py:198]: Remove code that's not required?

**Review by ExC0d3 — COMMENTED** [review]
LGTM.

**Review by gcallah — APPROVED** [review]
Things are working.

**Comment by gcallah** [comment]
Looks good to merge.

**Pull request merged**